app/src/main/java/com/mobile/tandil/kotlinbaseproject/activity/ParkingStayActivity.kt

```diff
@@ -0,0 +1,86 @@
+package com.mobile.tandil.kotlinbaseproject.activity
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mobile.tandil.kotlinbaseproject.databinding.ActivityParkingStayBinding
+import com.mobile.tandil.kotlinbaseproject.fragment.DatePickerDialogFragment
+import com.mobile.tandil.kotlinbaseproject.fragment.TimePickerDialogFragment
+import com.mobile.tandil.kotlinbaseproject.mvvm.viewmodel.ParkingStayViewModel
+import com.mobile.tandil.kotlinbaseproject.mvvm.viewmodel.ParkingStayViewModel.CheckState
+import com.mobile.tandil.kotlinbaseproject.mvvm.viewmodel.ParkingStayViewModel.ParkingStayData
+
+class ParkingStayActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityParkingStayBinding
+    private val viewModel = ParkingStayViewModel()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityParkingStayBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        viewModel.getValue().observe({ lifecycle }, ::updateUI)
+
+        setListeners()
+    }
+
+    private fun setListeners() {
+        binding.startDateTimeButton.setOnClickListener {
+            viewModel.showStartTimePickerFragment()
+            viewModel.showStartDatePickerFragment()
```

Comment:
you should only call the showStartDatePickerFragment method, then save the value, post another livedata state and there start the showStartTimePickerFragment method